gtda/mapper/nerve.py

```diff
@@ -130,22 +130,28 @@ def fit_transform(self, X, y=None):
         """
         # TODO: Include a validation step for X
         # Graph construction -- vertices with their metadata
-        nodes = reduce(iconcat, X, [])
-        graph = ig.Graph(len(nodes))
-
-        # Since `nodes` is a list, say of length N, of triples of the form
-        # (pullback_set_label, partial_cluster_label, node_elements),
-        # zip(*nodes) generates three tuples of length N, each corresponding to
-        # a type of node attribute.
-        node_attributes = zip(*nodes)
-        attribute_names = ["pullback_set_label", "partial_cluster_label",
-                           "node_elements"]
-        for i, node_attribute in enumerate(node_attributes):
-            graph.vs[attribute_names[i]] = node_attribute
+        nodes_dict = {}
```

Comment:
Perhaps we can be a bit clearer here by saying what the mapping is in the name of variable, e.g. `labels2nodes`? By itself, `nodes_dict` doesn't tell the reader much and including the type in the variable name is generally not considered best practice (even though I know I do it sometimes!)

Reply:
Thanks for this! I'll give it a better name along these lines.